app/src/androidTest/java/com/hedvig/app/feature/home/ChangeAddressButton.kt

```diff
@@ -0,0 +1,47 @@
+package com.hedvig.app.feature.home
```

Comment:
Make sure to format your files before committing them 🙂

Reply:
Damn, sorry. Would it be valuable to automate this somehow?

Reply:
I'm using Save Actions, but it's probably also possible to add some kind of pre-commit hook? 🤔